### DISCUSSION TITLE
Copy all scripts and paste them on to another sprite

### DISCUSSION THREAD

**Discussion by LilSefGaming** [ScratchAddons/ScratchAddons#4345] [Ideas]
Pls add another update, which I will now say, Copy all scripts and paste them on to another sprite, what I mean by copy is copying all scripts all at once!

**Comment by spacerhero132** [comment]
like copying all separate scripts

**Comment by Samq64** [comment]
Why can't you just duplicate the sprite?

**Reply by BroJac5246** [reply]
Probably if the sprite already had scripts + assets and you don't want to have to transfer all of them, or if you have to merge two sprites.

I naturally feel like Ctrl + A should select all of the blocks that could then be dragged. This would also include Ctrl-clicks and maybe Shift-clicks. It could be difficult to implement, though.
